Subject: Allow syscalls make by libfakeroot
Upstream-Author: Christoph Biedl <debian.axhn@manchmal.in-ulm.de>
Last-Update: 2019-07-22
Bug-Debian: https://bugs.debian.org/932762

--- a/src/seccomp.c
+++ b/src/seccomp.c
@@ -181,6 +181,7 @@
 #ifdef __NR_getdents64
 	ALLOW_RULE(getdents64);
 #endif
+	ALLOW_RULE(getpid);
 #ifdef FIONREAD
 	// called in src/compress.c under sread
 	ALLOW_IOCTL_RULE(FIONREAD);
@@ -198,6 +199,9 @@
  	ALLOW_RULE(mmap2);
 	ALLOW_RULE(mprotect);
 	ALLOW_RULE(mremap);
+	ALLOW_RULE(msgget);
+	ALLOW_RULE(msgrcv);
+	ALLOW_RULE(msgsnd);
 	ALLOW_RULE(munmap);
 #ifdef __NR_newfstatat
 	ALLOW_RULE(newfstatat);
@@ -211,6 +215,8 @@
 	ALLOW_RULE(rt_sigprocmask);
 	ALLOW_RULE(rt_sigreturn);
 	ALLOW_RULE(select);
+	ALLOW_RULE(semget);
+	ALLOW_RULE(semop);
 	ALLOW_RULE(stat);
 	ALLOW_RULE(stat64);
 	ALLOW_RULE(sysinfo);
